lib/src/components/NxTextLink/NxTextLink.tsx

```diff
@@ -47,7 +47,7 @@ const NxTextLink = forwardRef<HTMLAnchorElement, Props>(
           target = specifiedTarget || (newTab || (external && newTab !== false) ? '_blank' : ''),
           wrappedChildren = truncate ?
             <span className="nx-text-link__truncate-text">{children}</span> : ensureStartEndElements(children),
-          icon = <NxFontAwesomeIcon icon={faExternalLinkAlt} size="sm" />;
+          icon = <NxFontAwesomeIcon className="nx-icon--colorful" icon={faExternalLinkAlt} size="sm" />;
```

Comment:
What's the purpose of this change?  I guess it would affect external links within `NxTree`s, but I'm wondering if there's something else that caused you to add this.

Reply:
> What's the purpose of this change? I guess it would affect external links within `NxTree`s, but I'm wondering if there's something else that caused you to add this.

It was also for the `NxIconDropdown` external links, but it isn't needed anymore so I removed it here: 21c1ad2f8281c16939fc5afecf905965acb90e69

Reply:
> > It was also for the NxIconDropdown external links, but it isn't needed anymore so I removed it here: [21c1ad2](https://github.com/sonatype/sonatype-react-shared-components/commit/21c1ad2f8281c16939fc5afecf905965acb90e69)
> 
> Well I do think it was a desirable change, for external links in trees. In fact lets go ahead and add such a thing to one of the tree examples to confirm whether it would be needed for that to work correctly.

I added the example here: d4c760e166fb8d517b09f2a8e2940685fe1e95b5
There was some css that was already setting the color of all icons for links, but it was not using the `nx-icon--colorful` class so I removed it and added the className to the icon.